src/igvfd/types/modification.py

```diff
@@ -24,6 +29,9 @@ class Modification(Item):
         Path('lab', include=['@id', 'title']),
         Path('submitted_by', include=['@id', 'title']),
     ]
+    rev = {
+        'samples_modified': ('Sample', 'modifications')
```

Comment:
modifications is on the level of Biosample so it should link from there instead.